SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/ComponentIdGenerator.h

```diff
@@ -4,28 +4,37 @@
 
 struct FComponentIdGenerator
 {
-	FComponentIdGenerator(int StartId) : InitialId(StartId), IdReturned(StartId), NumIds(0)
-	{
-	}
+	const uint32 RESERVED_COMPONENT_ID_START = 19000;
```

Comment:
From the docs:
> Component IDs below 100 and from 190000 to 199999 are reserved for Improbable use.

Can you exclude this range as well?

Reply:
The 190000 to 199999 range is no longer reserved (docs update incoming I think). Will add a < 100 check.